tests/unit/h/services/annotation_stats_test.py

```diff
@@ -1,160 +1,114 @@
-from unittest import mock
+from unittest.mock import call, sentinel
 
 import pytest
 
-from h.search import (
-    DeletedFilter,
-    Limiter,
-    Search,
-    TopLevelAnnotationsFilter,
-    UserFilter,
-)
 from h.services.annotation_stats import AnnotationStatsService, annotation_stats_factory
 
 
 class TestAnnotationStatsService:
-    def test_total_user_annotation_count_calls_search_with_request(
-        self, svc, search, pyramid_request
+    def test_user_annotation_count(
+        self, pyramid_request, svc, Search, TopLevelAnnotationsFilter
     ):
-        svc.total_user_annotation_count("userid")
+        num_annotations = svc.user_annotation_count(sentinel.userid)
 
-        search.assert_called_with(pyramid_request)
```

Comment:
The tests for each method were split up into several separate tests: one for each assertion. But the actual method call and other setup is the same each time. I changed it to just one test with several assertions, which is what we'd normally do.